_sass/minima/_layout.scss

```diff
@@ -51,6 +55,14 @@
     border-radius: 5px;
     text-align: right;
 
+    label[for="nav-trigger"] {
+      display: block;
+      float: right;
+      width: 36px;
+      height: 36px;
+      z-index: 2;
```

Comment:
The hamburger menu appears in thin desktop windows as well. But perhaps it shouldn't be put here – I don't know how the SCSS media queries work.